README.md

```diff
@@ -105,5 +105,27 @@ Another option is to make a request directly to `/articles` which will give you
 
 The username/pw used for authentication in this application are openshift/secret.
 
+### Hot Deploy
+
+In order to dynamically pick up changes made in your application source code, you need to set the `RAILS_ENV=development` parameter to the [oc new-app](https://docs.openshift.org/latest/cli_reference/basic_cli_operations.html#basic-cli-operations) command, while performing the [installation steps](https://github.com/openshift/rails-ex#installation) described in this README.
+
+		$ oc new-app openshift/templates/rails-postgresql.json -p RAILS_ENV=development
+
+To change your source code in the running container you need to [oc rsh](https://docs.openshift.org/latest/cli_reference/basic_cli_operations.html#troubleshooting-and-debugging-cli-operations) into it.
+
+		$ oc rsh <POD_ID>
+
+After you [oc rsh](https://docs.openshift.org/latest/cli_reference/basic_cli_operations.html#troubleshooting-and-debugging-cli-operations) into the running container, your current directory is set to `/opt/app-root/src`, where the source code is located.
+
+To set your application back to the `production` environment you need to:
+
+* remove `RAILS_ENV` environment variable
+
+		$ oc env dc/rails-postgresql-example RAILS_ENV-
+
+* redeploy your application
```

Comment:
deploy will happen automatically, if it doesn't it's a bug:
https://github.com/openshift/rails-ex/blob/master/openshift/templates/rails-postgresql.json#L148